client/packages/ecore-tree-editor/src/browser/style/ecore-tree-editor.css

```diff
@@ -0,0 +1,85 @@
+.theia-tree-editor-form {
+    display: flex;
+}
+
+.theia-tree-editor-form>.jsonforms-container {
+    flex-grow: 1;
+    max-width: 960px;
+    /* Half the width of a full hd display. */
+    border-left: lightgray;
+    border-left-width: 1px;
+    border-left-style: solid
+}
+
+#ecore-tree-editor>div>div.p-Widget.theia-tree-editor-form.p-SplitPanel-child>div>div>div:nth-child(1) {
+    text-transform: uppercase;
+    font-size: var(--theia-ui-font-size2);
+    margin: 10px 5px;
+}
+
+.control>label {
+    text-transform: uppercase;
+    font-size: var(--theia-ui-font-size1) !important;
+}
+
+.theia-tree-editor-tree .theia-ExpansionToggle {
+    margin-right: -4px;
+}
+
+.ecoreimg {
+    background-size: 16px 16px;
+    height: 16px;
+    width: 16px;
+    margin-right: 6px;
+}
+
+.eattribute {
+    background-image: url(../../../../sprotty-ecore/images/EAttribute.gif);
```

Comment:
These long relative URIs feel a bit fragile. Can we improve this, e.g. by sharing resources into a different package?

Reply:
That's a good idea! I moved them to a new common package and also moved the css classes there, as they are used by both the command palette and the tree editor.